packages/sanity/src/structure/comments/src/components/list/CommentThreadLayout.tsx

```diff
@@ -129,7 +131,7 @@ export function CommentThreadLayout(props: CommentThreadLayoutProps) {
       </HeaderFlex>
 
       {canCreateNewThread && (
-        <ThreadCard onClick={handleNewThreadClick} tone={isSelected ? 'primary' : undefined}>
+        <ThreadCard onClick={handleNewThreadClick} tone={isSelected ? 'caution' : undefined}>
```

Comment:
Just to confirm, is this change intentional?

Reply:
Yes, it is intentional!